grpclib/client.py

```diff
@@ -752,10 +753,11 @@ def _get_default_ssl_context(self) -> '_ssl.SSLContext':
         ctx.options |= (_ssl.OP_NO_TLSv1 | _ssl.OP_NO_TLSv1_1)
         ctx.set_ciphers('ECDHE+AESGCM:ECDHE+CHACHA20:DHE+AESGCM:DHE+CHACHA20')
         ctx.set_alpn_protocols(['h2'])
-        try:
-            ctx.set_npn_protocols(['h2'])
-        except NotImplementedError:
-            pass
+        if ssl.HAS_NPN:
+            try:
+                ctx.set_npn_protocols(['h2'])
+            except NotImplementedError:
```

Comment:
We probably don't need this `try..except` clause anymore and it would be sufficient to only check `HAS_NPN` constant.